.travis.yml

```diff
@@ -20,6 +21,7 @@ matrix:
 before_install:
     - phpenv config-rm xdebug.ini || true
     - composer self-update
+    - if [ "$LINT" != 1 ]; then composer remove --dev --no-update phpstan/phpstan; fi;
```

Comment:
💡 This Projects locks phpstan at version 0.11 which is not compatible with php 8. You probably should bump to 0.12 eventually, but this seemed out of scope for this PR. This is why I chose to remove phpstan when it's not needed.